Packages/tests/HardwareBasic/UTF_BasicHardwareTests.ipf

```diff
@@ -1864,6 +1864,9 @@ static Function StopDAQDueToUncompiled_REENTRY([str])
 	CheckThatTestpulseRan_IGNORE(str)
 
 	CheckDAQStopReason(str, DQ_STOP_REASON_UNCOMPILED)
+
+	// remove left over from ???
```

Comment:
The reason is the following:
You fill the execution queue with a command and then another command "KillVariables/Z V_flag".
Then after the first command is executed the background task is executed before "KillVariables/Z V_flag".
Thus, this solution is rather flaky for all the reentry testcases if there is an Execute/P/Q/Z in the first part.

Reply:
Change ForceCompile to not use /Z and remove above fixup.

Reply:
Fixed.